src/bot.coffee

```diff
@@ -217,8 +217,8 @@ class SlackBot extends Adapter
       id: user.id
       name: user.name
       real_name: user.real_name
-      email_address: user.profile.email
       slack: {}
+    newUser.email_address = user.profile.email unless not user.profile
```

Comment:
`unless not`

![fillion](https://cloud.githubusercontent.com/assets/32463/21786303/0ffae9fc-d678-11e6-82ca-4964a98c844d.gif)